examples/incremental_learning/helmet_detection/training/inference.py

```diff
@@ -100,10 +105,29 @@ def deal_infer_rsl(model_output):
 def run():
     camera_address = Context.get_parameters('video_url')
 
+    hard_example_name = Context.get_parameters('HEM_NAME', "IBT")
```

Comment:
How does a developer find the env 'HEM_NAME'?  If this is a sedna system environment varibles, it should provide a function or class for the developer, instead of expose an environment name which the developer might find only in documents.

In sedna, there are two seperated parameter entries for deployer and developer, and the parameter should be subject to CRD or LIB.

for example:

```
# Select one of the next two lines (ignore the name of function or varibles): 
ibt=IL.get_hem_from_crd()  # subject to CRD
ibt=IL.IBT(0.9,0.9)        # subject to LIB

instance = IL(
    estimator=e,
    hem=ibt
)```

Reply:
fix